duckduckgo/Android
# Fix/david/unprotected sites

Task/Issue URL: https://app.asana.com/0/488551667048375/1200764858669939
Tech Design URL: 
CC: 

**Description**:https://github.com/duckduckgo/Android/pull/1358 introduced an issue with Unprotected sites, making it look different to what we wanted. This PR fixes that issue

**Steps to test this PR**:
1. Open Unprotected sites
2. Check that the design is the same as other screens with lists (Fireproof sites, Bookmarks, etc...)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
